mapclassify/classifiers.py

```diff
@@ -1520,12 +1520,13 @@ class StdMean(MapClassifier):
 
     Parameters
     ----------
-
-    y : numpy.array
-        :math:`(n,1)`, values to classify.
-    multiples : numpy.array (default [-2, -1, 1, 2])
-        The multiples of the standard deviation to add/subtract from
-        the sample mean to define the bins
+    y         : array
```

Comment:
I wondering if we can keep the docstrings in the current style? I spent quite of bit of time combing through these for standard formatting, punctuation, grammar, etc.? I have added the `anchor` entry in that style here.

```suggestion
y : numpy.array
    :math:`(n,1)`, values to classify.
multiples : numpy.array (default [-2, -1, 1, 2])
    The multiples of the standard deviation to add/subtract from the sample mean to define the bins.
anchor : bool (default False)
    Anchor upper bound of one class to the sample mean.
```

Reply:
Agree with @jGaboardi. What we have now follows the [numpydoc guide](https://numpydoc.readthedocs.io/en/latest/format.html), the change does not and requires reformatting the whole thing in case you add a new keyword longer than the previously longest one.